util/docker/maketest.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Runs tests and reports on them to github
+
+DIR=$(cd $(dirname $0) ; pwd -P)
+source ${DIR}/docker_common.sh
+
+# Get sha sum
+USER="jgkamat"
```

Comment:
Yeah we might as well use georgeburdell.  I actually entered an environment variable GH_USER (set to georgeburdell) that you can use for this.